src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddCaseNoteCallbackHandler.java

```diff
@@ -61,11 +63,19 @@ private CallbackResponse moveNotesIntoList(CallbackParams callbackParams) {
         CaseData updatedCaseData = caseData.toBuilder()
             .caseNotes(caseNotes)
             .caseNote(null)
-            .businessProcess(BusinessProcess.ready(ADD_CASE_NOTE))
```

Comment:
is it not going to use Camunda Process?

Reply:
No, the whole point of this ticket is to remove from camunda